client/state/jetpack/modules/reducer.js

```diff
@@ -2,7 +2,7 @@
  * External dependencies
  */
 import { combineReducers } from 'redux';
-import { forEach, merge, pickBy } from 'lodash';
+import { forEach, merge, pickBy, get } from 'lodash';
```

Comment:
Totally nitpicking here: 

I'd alter this to `{ forEach, get, merge, pickBy }` to maintain the alphabetical order of imported methods.